--- origsrc/libQTidy-0.0.14/QTidy.pro	2008-07-29 15:13:05.000000000 -0500
+++ src/libQTidy-0.0.14/QTidy.pro	2008-07-31 00:22:56.468750000 -0500
@@ -13,5 +13,3 @@
 	qtidygui
 
 data.files = ChangeLog COPYING COPYING.lib NEWS README
-data.path = $$[QT_INSTALL_PREFIX]/share/qtidy
-INSTALLS += data
--- origsrc/libQTidy-0.0.14/qtidygui/qtidygui.pro	2008-07-29 15:13:05.000000000 -0500
+++ src/libQTidy-0.0.14/qtidygui/qtidygui.pro	2008-07-31 00:24:11.843750000 -0500
@@ -48,10 +48,10 @@
 	## OpenSUSE Bugfix :-/
 	INCLUDEPATH += "/usr/include/tidy"
 
-	pixmap.path += $$[QT_INSTALL_PREFIX]/share/pixmaps
+	pixmap.path += /usr/share/pixmaps
 	pixmap.files += qtidyrc.png
 
-	xdg.path += $$[QT_INSTALL_PREFIX]/share/applications
+	xdg.path += /usr/share/applications
 	xdg.files += qtidyrc.desktop
 
 	debug {
@@ -67,7 +67,7 @@
 
 }
 
-target.path += $$[QT_INSTALL_BINS]
+target.path += /usr/bin
 
 INSTALLS += target \
 	translations
