--- origsrc/libQTidy-0.2.0/src/pkgconfig/QTidy.pc	2008-09-26 08:28:08.000000000 -0500
+++ src/libQTidy-0.2.0/src/pkgconfig/QTidy.pc	2008-09-28 01:35:13.668244400 -0500
@@ -1,11 +1,11 @@
 prefix=@QTIDY_PREFIX@
 exec_prefix=${prefix}
-libdir=${prefix}/lib
-includedir=${prefix}/include
+libdir=@QTIDY_LIBDIR@
+includedir=@QTIDY_INCLUDEDIR@
 
 Name: libQTidy
-Description: a qt4 library that implements tity functions
+Description: a qt4 library that implements tidy functions
 Version: @QTIDY_VERSION@
 Requires: @QTIDY_REQUIRE@
 Libs: -L${libdir} -lQTidy -ltidy
-Cflags: -I${includedir}/QTidy -I${includedir}/tidy
+Cflags: -I${includedir}/QTidy -I/usr/include/tidy
--- origsrc/libQTidy-0.2.0/src/pkgconfig.pri	2008-09-26 08:28:08.000000000 -0500
+++ src/libQTidy-0.2.0/src/pkgconfig.pri	2008-09-28 01:36:09.371369400 -0500
@@ -1,12 +1,16 @@
 CONFIG += create_pc explicitlib
 QMAKE_PKGCONFIG_DESTDIR = pkgconfig
 QMAKE_PKGCONFIG_NAME = QTidy
-QMAKE_PKGCONFIG_DESCRIPTION = a qt4 library that implements tity functions
-QMAKE_PKGCONFIG_REQUIRES = QtCore
+QMAKE_PKGCONFIG_DESCRIPTION = a qt4 library that implements tidy functions
+QMAKE_PKGCONFIG_REQUIRES = QtCore QtGui QtNetwork QtXml
 prefix_replace.match = @QTIDY_PREFIX@
 prefix_replace.replace = $$[QT_INSTALL_PREFIX]
 version_replace.match = @QTIDY_VERSION@
 version_replace.replace = $${VERSION}
 requires_replace.match = @QTIDY_REQUIRE@
 requires_replace.replace = $${QMAKE_PKGCONFIG_REQUIRES}
-QMAKE_PKGCONFIG_INSTALL_REPLACE += prefix_replace version_replace requires_replace
+includedir_replace.match = @QTIDY_INCLUDEDIR@
+includedir_replace.replace = $$[QT_INSTALL_HEADERS]
+libdir_replace.match = @QTIDY_LIBDIR@
+libdir_replace.replace = $$[QT_INSTALL_LIBS]
+QMAKE_PKGCONFIG_INSTALL_REPLACE += prefix_replace version_replace requires_replace includedir_replace libdir_replace
