--- origsrc/libQTidy-0.2.0/qtidygui/qtidygui.pro	2008-09-26 08:28:08.000000000 -0500
+++ src/libQTidy-0.2.0/qtidygui/qtidygui.pro	2008-09-28 01:15:54.746369400 -0500
@@ -72,7 +72,7 @@
 	INSTALLS += xdg pixmap
 }
 
-target.path += $$[QT_INSTALL_BINS]
+target.path += $$bin.path
 
 INSTALLS += target \
 	translations \
--- origsrc/libQTidy-0.2.0/qtidyxml/qtidyxml.pro	2008-09-26 08:28:08.000000000 -0500
+++ src/libQTidy-0.2.0/qtidyxml/qtidyxml.pro	2008-09-28 01:15:59.527619400 -0500
@@ -26,12 +26,13 @@
 LIBS += -L../src -lQTidy
 
 include(../flags.pri)
+include(../config.pri)
 
 HEADERS =	qtidyxml.h
 
 SOURCES = main.cpp \
 	qtidyxml.cpp
 
-target.path += $$[QT_INSTALL_BINS]
+target.path += $$bin.path
 
 INSTALLS += target
